core/src/main/scala/chisel3/Layer.scala

```diff
@@ -51,6 +63,31 @@ object layer {
       case _          => s"${parent.fullName}.$name"
     }
 
+    /**
+      * The output directory of this layer.
```

Comment:
```suggestion
    /** The output directory of this layer.
```
Officially in ScalaDoc you put the "short comment" on the first line. This is package private so won't show up in ScalaDoc anyway but I still think it's good practice.